src/lib/portal-api.ts

```diff
@@ -82,6 +81,19 @@ export const getPortalOfferings = (
         }
       });
     }
+    else if (userType === "learner" && offeringId) {
```

Comment:
It is possible some code is looking for the existence of the portalOfferings to decide about showing some teacher only things. I hope that isn't the case, but just wanted to point it out in case this change causes bugs for students.

The upshot is that a comment on the story should be added about making sure to test this with portal launching by students.